### PR TITLE
Add GELF TCP handler and disable compression by default

### PR DESCRIPTION
**Pull request by mlosev**
Allows to use TCP handler for reliable logs delivery

Also disabling compression by default to allow use proxy like https://github.com/jedisct1/flowgger

### CODE REVIEW

**Comment by gredin** [comment]
The option for disabling the compression is a must have :+1:

**Comment by severb** [comment]
I think disabling compression will change the behaviour for current users. I think it should be enabled by default.

**Comment by gredin** [comment]
@mlosev I think @severb has a point. Are you going to change the default behaviour or may I do it?

File: graypy/handler.py [hunk 1]
```diff
@@ -62,7 +64,49 @@ def makePickle(self, record):
...
+        self.debugging_fields = debugging_fields
+        self.extra_fields = extra_fields
+        self.fqdn = fqdn
+        self.localname = localname
+        self.facility = facility
+        self.level_names = level_names
+        self.compress = compress
+        SocketHandler.__init__(self, host, port)
+
+    def makePickle(self, record):
```
Comment (tusharmakkar08) [inline graypy/handler.py]: Can we DRY this? Maybe make a class and then make it inherited both by `GELFHandler` and  `GELFTcpHandler` .

**Comment by tusharmakkar08** [comment]
@mlosev @severb : Is this PR active? I think this is an important feature for Graypy.

**Comment by severb** [comment]
I think this PR was blocked by another change that was landed in the meantime: #60. It should be ready to land after the changes you requested.

**Comment by tusharmakkar08** [comment]
Fixed [here](https://github.com/severb/graypy/pull/73)

**Pull request closed without merging**